app/shared/video-player/common/ui/VideoControllerState.kt

```diff
@@ -11,29 +12,66 @@ import androidx.compose.runtime.snapshots.SnapshotStateList
 
 
 /**
- * @param initialVisible 变更不会更新
+ * @param initialVisibility 变更不会更新
  */
 @Composable
 fun rememberVideoControllerState(
-    initialVisible: Boolean = false
+    initialVisibility: ControllerVisibility = ControllerVisibility.Invisible
 ): VideoControllerState {
     return remember {
-        VideoControllerState(initialVisible)
+        VideoControllerState(initialVisibility)
     }
 }
 
+@Immutable
+data class ControllerVisibility(
+    val topBar: Boolean,
+    val bottomBar: Boolean,
+    val floatingBottomEnd: Boolean,
+    val rhsBar: Boolean,
+    val detachedSlider: Boolean
+) {
+    companion object {
+        val Visible = ControllerVisibility(
```

Comment:
```suggestion
        @Stable
        val Visible = ControllerVisibility(
```
下面几个同理. 如果这个东西会在 composable 代码里面使用到, 增加 `@Stable` 可以轻微优化性能